lib/WebLedgerClient.js

```diff
@@ -292,33 +302,34 @@ class WebLedgerClient {
    */
   async getAgents({headers = {}} = {}) {
     const baseURL = `https://${this.hostname}`;
-    const requestHeaders = Object.assign(
-      {}, constants.DEFAULT_HEADERS, headers);
-    const agentsApi =
-      create({baseURL, httpsAgent: this.httpsAgent, timeout: this.timeout});
-    const response = await agentsApi.get('/ledger-agents', {}, {
-      headers: requestHeaders
-    });
-    if(response.problem) {
+    try {
+      const {data} = await httpClient.get('ledger-agents', {
+        prefixUrl: baseURL,
+        headers: {...DEFAULT_HEADERS, ...headers},
+        agent: this.httpsAgent,
+        timeout: this.timeout
+      });
```

Comment:
Please make the use of the httpClient consistent across all API calls here in this library, as well as the way it's being used in the reset of the DB stack.  The fact that the `prefixUrl` features bears some resemblance to the old `apisauce` API is immaterial. We need all DB developers to be able to look at this code and recognize the `http-client` usage when doing code review and modifying the code.

```suggestion
      const {data} = await httpClient.get(`${baseUrl}/ledger-agents`, {
        headers: {...DEFAULT_HEADERS, ...headers},
        agent: this.httpsAgent,
        timeout: this.timeout
      });
```